src/rom_header_rhh.c

```diff
@@ -21,6 +22,8 @@ struct RHHRomHeader
     /*0x10*/ u32 numSpecies;
     /*0x14*/ u32 abilitiesCount;
     /*0x18*/ const struct Ability *abilities;
+    /*0x1C*/ u32 itemsCount;
+    /*0x20*/ u8 itemNameLength;
```

Comment:
To make the most out of this space, let's add some padding here for other `u8`s
```suggestion
    /*0x20*/ u8 itemNameLength;
    /*0x21*/ u8 padding[3];
```